charts/plugin-br-bank-transfer/templates/configmap.yaml

```
@@ -7,6 +7,8 @@ metadata:
 data:
   # Application Settings
   ENV_NAME: {{ .Values.bankTransfer.configmap.ENV_NAME | default "production" | quote }}
+  DEPLOYMENT_MODE: {{ .Values.bankTransfer.configmap.DEPLOYMENT_MODE | default "byoc" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)